rasa_core/policies/embedding_policy.py

```diff
@@ -269,18 +269,18 @@ def _load_visual_params(self, config: Dict[Text, Any]) -> None:
         self.evaluate_every_num_epochs = config['evaluate_every_num_epochs']
         if self.evaluate_every_num_epochs < 1:
             self.evaluate_every_num_epochs = self.epochs
-
         self.evaluate_on_num_examples = config['evaluate_on_num_examples']
 
     def _load_params(self, **kwargs: Dict[Text, Any]) -> None:
-        config = copy.deepcopy(self.defaults)
-        config.update(kwargs)
+        self.config = copy.deepcopy(dict(self.defaults, **self.tf_defaults))
```

Comment:
since we persist only tf_config now in both policies, no need for self. here. We needed it in previous iterations where the whole config was persisted
```suggestion
        config = copy.deepcopy(dict(self.defaults, **self.tf_defaults))
```